test/parallel/test-buffer-alloc.js

```diff
@@ -768,14 +768,8 @@ Buffer.from(Buffer.allocUnsafe(0), 0, 0);
 }
 
 // issue GH-4331
-assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFF),
- common.engineSpecificMessage({
-   v8: RangeError,
-   chakracore: TypeError}));
-assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFFF),
- common.engineSpecificMessage({
-   v8: RangeError,
-   chakracore: TypeError}));
+assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFF));
```

Comment:
This is because https://github.com/Microsoft/ChakraCore/commit/9b716d57cc1bb9c47640ad10260c3a084f6d9b9a changed the behavior. Note that `chakracore-master` will not reflect this behavior because it is still on 1.3